src/Action_Closest.cpp

```diff
@@ -318,11 +317,25 @@ Action::RetType Action_Closest::DoAction(int frameNum, ActionFrame& frm) {
       V_atom_coords_[index + 2] = a[2];
     }
   }
-  // TODO handle imaging
+  
+
+  //low overhead so we will do it anyway
+  Box frmBox = frm.Frm().BoxCrd();
+  Matrix_3x3 ucell, recip;
+  frm.Frm().BoxCrd().ToRecip(ucell, recip);
+  double box[3] = {frmBox[0], frmBox[1],frmBox[2]};
```

Comment:
Ah! I was looking for that. I was not aware that we had access to the underlying pointers. I will make the change.